src/ML-SNAP/compute_snap.cpp

```diff
@@ -208,20 +228,21 @@ ComputeSnap::ComputeSnap(LAMMPS *lmp, int narg, char **arg) :
 
 ComputeSnap::~ComputeSnap()
 {
+
```

Comment:
Please undo the whitespace changes here. We want to keep the programming style as consistent as possible throughout LAMMPS to make it easier for people to read and modify. That sometimes requires to against personal preferences.